pkg/recipes/terraform/config/config.go

```diff
@@ -150,13 +167,24 @@ func getProviderConfigs(ctx context.Context, requiredProviders []string, support
 			return nil, err
 		}
 		if len(config) > 0 {
-			providerConfigs[provider] = config
+			providerConfigs[provider] = []any{config}
 		}
 	}
 
 	return providerConfigs, nil
 }
 
+// getRecipeProviderConfigs returns the Terraform provider configurations for Terraform providers.
+func getRecipeProviderConfigs(ctx context.Context, envConfig *recipes.Configuration) map[string][]any {
```

Comment:
Since this is specific to providers, what do you think of moving this under terraform/config/providers package in [types.go](https://github.com/radius-project/radius/blob/12e1e49ba11fe387ad27630b0cd2d154c20f2700/pkg/recipes/terraform/config/providers/types.go)?

Reply:
moved getRecipeProviderConfigs() to terraform/config/providers/types.go